src/i18n/ES/grid-resources.ts

```diff
@@ -0,0 +1,76 @@
+import { IGridResourceStrings } from 'igniteui-angular';
+
+/**
+ * Spanish resource strings for IgxGrid
+ */
+export const GridResourceStringsES: IGridResourceStrings = {
+  igx_grid_groupByArea_message: 'Arrastre un encabezado de columna y suéltelo aquí para agrupar por esa columna.',
+  igx_grid_emptyFilteredGrid_message: 'No se encontraron Registros.',
+  igx_grid_emptyGrid_message: 'El Grid no tiene Datos.',
+  igx_grid_filter: 'Filtro',
+  igx_grid_filter_row_close: 'Cerrar',
+  igx_grid_filter_row_reset: 'Reiniciar',
+  igx_grid_filter_row_placeholder: 'Añadir un Valor para el Filtro',
+  igx_grid_filter_row_boolean_placeholder: 'Todos',
+  igx_grid_filter_row_date_placeholder: 'Fecha de Recogida',
+  igx_grid_filter_operator_and: 'Y',
+  igx_grid_filter_operator_or: 'O',
+  igx_grid_complex_filter: 'Filtro Complejo',
+  igx_grid_filter_contains: 'Contiene',
+  igx_grid_filter_doesNotContain: 'No Contiene',
+  igx_grid_filter_startsWith: 'Empieza Con',
+  igx_grid_filter_endsWith: 'Termina Con',
+  igx_grid_filter_equals: 'es Igual',
+  igx_grid_filter_doesNotEqual: 'No es Igual',
+  igx_grid_filter_empty: 'Vacío',
+  igx_grid_filter_notEmpty: 'No Vacío',
+  igx_grid_filter_null: 'Nulo',
+  igx_grid_filter_notNull: 'No Nulo',
+  igx_grid_filter_before: 'Antes',
+  igx_grid_filter_after: 'Despues',
+  igx_grid_filter_today: 'Hoy',
+  igx_grid_filter_yesterday: 'Ayer',
+  igx_grid_filter_thisMonth: 'Este Mes',
+  igx_grid_filter_lastMonth: 'El Mes Pasado',
+  igx_grid_filter_nextMonth: 'El Próximo Mes',
+  igx_grid_filter_thisYear: 'Este Año',
+  igx_grid_filter_lastYear: 'El Año Pasado',
+  igx_grid_filter_nextYear: 'El Próximo Año',
+  igx_grid_filter_greaterThan: 'Mayor que',
+  igx_grid_filter_lessThan: 'Menor que',
+  igx_grid_filter_greaterThanOrEqualTo: 'Mayor o Igual que',
+  igx_grid_filter_lessThanOrEqualTo: 'Menor o igual que',
+  igx_grid_filter_true: 'Verdadero',
+  igx_grid_filter_false: 'Falso',
+  igx_grid_filter_all: 'Todas',
+  igx_grid_filter_condition_placeholder: 'Seleccionar Filtro',
+  igx_grid_summary_count: 'Cuenta',
+  igx_grid_summary_min: 'Min',
+  igx_grid_summary_max: 'Max',
+  igx_grid_summary_sum: 'Sum',
+  igx_grid_summary_average: 'Promed',
+  igx_grid_summary_earliest: 'El mas temprano',
+  igx_grid_summary_latest: 'El último',
+  igx_grid_excel_filter_moving_left: 'izquierda',
+  igx_grid_excel_filter_moving_right: 'derecha',
+  igx_grid_excel_filter_moving_header: 'moviendo columna',
```

Comment:
This can be seen in the Column options in [the Excel example](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel_style_filtering.html).
It think "Mover columna" is better suited for the use context.